interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
I think @mhuyck was concerned about the performance of hasOwnProperty, so I didn't comment on the other spots. I only commented on the one that appeared to be an array - that should not cause a performance issue and avoids the danger of iterating over non-numeric properties.